Docs/Particles/Particles.tex

```diff
@@ -1,215 +1,149 @@
+	
+\subsection{randomly-assigned initial positions}
+\label{particle:randomplacement}
+Alternatively, the positions of particles can be randomly chosen. This may be used to characterize the global movement of a fluid. Even in this way of placing particles, however, the trajectory of each particle can be traced using the index individually assigned to each particle, which can be found in the output file (see Section \ref{particles:output_file}).
 
-\noindent To enable this option, set \\
-
-\noindent {\bf castro.particle\_move\_type = Gravitational} \\
+To enable this option, set \\
 
-\subsubsection{Move-Kick-Drift Algorithm}
+\noindent {\tt {\bf castro.castro.particle\_init\_type = Random}}. \\
 
-In each time step:
-\begin{itemize}
-\item Solve for ${\mathbf g}^n$  (only if multilevel, otherwise use ${\mathbf g}^{n+1}$ from previous step)
-\item ${\mathbf u}_i^{\nph} = \frac{1}{a^{\nph}} ( (a^n {\mathbf u}^n_i) + \frac{\dt}{2} \; {\mathbf g}^n_i )$
-\item ${\mathbf x}_i^{n+1 } = {\mathbf x}^n_i +  \frac{\dt}{a^{\nph}}  {\mathbf u}_i^{\nph}$
-\item Solve for ${\mathbf g}^{n+1}$ using ${\mathbf x}_i^{n+1}$
-\item ${\mathbf u}_i^{n+1} = \frac{1}{a^{n+1}} ( (a^{\nph} {\mathbf u}^{\nph}_i) + \frac{\dt}{2} \; {\mathbf g}^{n+1}_i )$
-\end{itemize}
+\noindent The number of particles are determined by setting: \\
 
-Note that at the end of the timestep ${\bf x}_i^{n+1}$ is consistent with ${\bf g}^{n+1}$ becasue
-we have not advanced the positions after computing the new-time gravity.  This has the benefit that
-we perform only one gravity solve per timestep (in a single-level calculation with no hydro) because
-the particles are only moved once.
+\noindent {\tt {\bf castro.particle\_initrandom\_count = }} particle \# (e.g., 10000).
 
-\subsubsection{Computing {\bf g}}
 
-We solve for the gravitational vector as follows:
-\begin{itemize}
-\item Assign the mass of the particles onto the grid in the form of density, $\rho_{DM}$.  
-The mass of each particle is assumed to be uniformly distributed over a cube of side $\Delta x$, 
-centered at what we call the position of the particle.    We distribute the mass of each
-particle to the cells on the grid in proportion to the volume of the intersection of each cell
-with the particle's cube.   We then divide these cell values by $\Delta x^3$ so that the
-right hand side of the Poisson solve will be in units of density rather than mass.  
-Note that this is the {\it comoving} density.
+\section{Particle movement}
+%I did not remove this part just in case this is implemented in the code. But if not, please remove it.
+\noindent There are currently two different ways in which particles can be moved, random motion and motion by self-gravity.
```

Comment:
It looks like these methods don't exist.  If you look in `CastroParticles.cpp` it looks like it just advects wiith the cell-centered velocity:
```
TracerPC->AdvectWithUcc(Ucc, level, dt); 
```

Reply:
I got it. I was suspicious about this section, especially "motion by gravity", since I think particles are considered as massless particles in CASTRO. I will remove that part and add some more as I play with this. Then I will do the PR.